src/components/Facility/Consultations/VentilatorPlot.tsx

```
@@ -28,44 +26,194 @@
-    };
+    const { graphData } = getGraphData(dailyRoundsList);
+    setResults(graphData);
+  }, [dailyRoundsList]);
+
+  const getGraphData = (dailyRoundsData?: DailyRoundsModel[]) => {
+    let graphData = {};
+    const graphDataCount = dailyRoundsData?.length ?? 0;
+    if (dailyRoundsData) {
+      graphData = dailyRoundsData.reduce(
+        (acc, currentRound: DailyRoundsModel) => ({
+          ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)